### PR TITLE
Create CUBiDA_colloquium_Erlangen_Introduction_to_Galaxy.md

### PR DESCRIPTION
**Pull request by PGbioinf**
Hi,

This is Pooja Gupta from Erlangen. I will be giving a talk about Galaxy tomorrow in our monthly colloquium in Erlangen. @bgruening Bjoern suggested that I can create an event/post about this talk so that it appears here: https://galaxyproject.eu/news . Therefore, I am creating this pull request with a description of the event.

I will be grateful if this event can be listed.
Thank you. 
Pooja

### CODE REVIEW

**Review by beatrizserrano — APPROVED** [review]
Thank you very much @PGbioinf, I'll merge it now and will be online soon :) Good luck with the talk!

File: _events/CUBiDA_colloquium_Erlangen_Introduction_to_Galaxy.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- Tutorial
+title: "Introduction to Galaxy - a web-based platform for bioinformatics analysis"
+starts: 2021-06-23
+ends: 2021-06-23
+organiser: Pilot Core Unit Bioinformatics, Data Integration and Analysis, Universitätklinikum Erlangen
+location: online (https://uk-erlangen.webex.com/uk-erlangen/j.php?MTID=mfa8088026974da77de04e981fb3b7abd)
+supporters: 
+ - galaxy-europe
+ - galaxy team freiburg
+ - Life@FAU
```
Comment (bgruening) [inline _events/CUBiDA_colloquium_Erlangen_Introduction_to_Galaxy.md:14]: @PGbioinf do you think we should/could add MIRACUM here?

Comment (PGbioinf) [reply _events/CUBiDA_colloquium_Erlangen_Introduction_to_Galaxy.md:14]: yes, thank you so much Bjoern!

Comment (beatrizserrano) [reply _events/CUBiDA_colloquium_Erlangen_Introduction_to_Galaxy.md:14]: I'll do it :)

Comment (PGbioinf) [reply _events/CUBiDA_colloquium_Erlangen_Introduction_to_Galaxy.md:14]: > 
> 
> I'll do it :)

Thank you so much!

**Pull request merged**